README.md

```diff
@@ -26,7 +26,42 @@
 The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
 also a live [Google Calendar][] that may be subscribed to.
 
-# LTS Plan
+## Mandate
+
+The Release and LTS working group's purpose is:
+
+* Management/execution the release process for all releases.
+* Maintenance of the LTS releases.
+
+Its responsibilities are:
+
+* Define the release process.
+* Generate and create releases. 
+* Manage the LTS and Current branches including backporting changes to 
+  these branches.
+
+The Release and LTS working group is structured into teams and membership in
+the working group does not automatically result in membership in these
+teams. These teams are:
+
+* Release team
+* LTS team
+* Backporting team
```

Comment:
Currently the LTS responsibilities fall into two categories:

1. Determining the LTS release schedule
2. Deciding what goes into the LTS releases

The backporting team handles item 2. Both the release team and backporting teams can share responsibility for 1, meaning that we really only need two teams: backporting and release, who work together on the schedule.